torchci/lib/bot/mergeBot.ts

```diff
@@ -47,14 +47,14 @@ function mergeBot(app: Probot): void {
         return;
       }
       const revertWithReasonCmdPat = new RegExp(
-        "^\\s*@pytorch(merge|)bot\\s+revert\\s+this.*(\\s+\\w+){3,}"
+        "^\\s*@pytorch(merge|)bot\\s+revert\\s+this(.|\s)*(\\s+\\w+){3,}"
```

Comment:
```suggestion
        "^\\s*@pytorch(merge|)bot\\s+revert\\s+this(.|\\s)*(\\s+\\w+){3,}"
```
I think \s just matches the letter s?

Reply:
oH YES good catch